Python/ibmcloudsql/SQLQuery.py

```diff
@@ -296,13 +290,12 @@ def list_results(self, jobId):
 
         fourth_slash = result_location.replace('/', 'X', 3).find('/')
 
-        response = self.client.fetch(
+        response = requests.get(
             result_location[:fourth_slash] + '?prefix=' + result_location[fourth_slash + 1:],
-            method='GET',
-            headers=self.request_headers,
-            validate_cert=False)
+            params=self.request_headers,
```

Comment:
Should this end with a , after params?

Reply:
Good question. short answer is yes, trailing comma is recommended for multi-line parameters. See [When to Use Trailing Commas](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)

and practice https://stackoverflow.com/a/11597911/914967

> The main advantages are that it makes multi-line lists easier to edit and that it reduces clutter in diffs.